source/ref_gl/r_math_util.c

```diff
@@ -0,0 +1,66 @@
+#include "r_math_util.h"
+
+// sin values are generated from this: 
+// [math.sin((i/ 255.0) * 2 * math.pi) for i in  range(0,256)]
+// for( size_t i = 0; i < 256; i++ )
+//		rsh.sinTableByte[i] = sin( (float)i / 255.0 * M_TWOPI );
+static float sinTable[256] = {
```

Comment:
sinTable under rsh leaks details that shouldn't need to be exposed. would be interested to run this in perf and see how much this saves.